platform-sdk/swirlds-common/src/main/java/com/swirlds/common/io/streams/SerializableDataInputStream.java

```diff
@@ -54,7 +55,8 @@
  */
 public class SerializableDataInputStream extends AugmentedDataInputStream {
 
-    private static final int PROTOCOL_VERSION = SERIALIZATION_PROTOCOL_VERSION;
```

Comment:
this version refers to the serialization protocol version. the signed state should have a separate version number which should be used

Reply:
okay, I no longer use it for these purposes.